source/projects/manage-relational-connection.txt

```diff
@@ -9,13 +9,13 @@ Manage the Relational Model
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-You can use the :guilabel:`Manage` button on the :guilabel:`Schema model` pane to 
-make changes to the schemas or tables you want to include in 
+You can use the :guilabel:`Manage` button on the :guilabel:`Schema model` 
+pane to make changes to the schemas or tables you want to include in 
 your migration project. You can also connect to the source database 
```

Comment:
[question]

Should this be "sync" to be consistent with the other changes, or would that inadvertently change the meaning here? Thanks!

Reply:
I think "migration project" is adequate here. The UI refers to the concept as a "sync job", I think the entire project makes sense as a "migration project" that utilizes "sync jobs" for the data transfer.